src/ansys/units/__init__.py

```diff
@@ -23,7 +23,7 @@
 from ansys.units.map import QuantityMap, QuantityMapError  # noqa: F401
 from ansys.units.quantity import Quantity, QuantityError  # noqa: F401
 from ansys.units.systems import UnitSystem, UnitSystemError  # noqa: F401
-from ansys.units.units import Units, parse_temperature_units  # noqa: F401
+from ansys.units.utils import *  # noqa: F401
```

Comment:
Yes, you want to avoid `import *`.  Instead you want to be explicit. eg:
```
from ansys.unts.utils import (
  function1,
  function2,
  ...
)
```

Reply:
That said, I think you should remove this import from here entirely and just explicitly import the functions as needed in the rest of the code.  I don't think they should be part of the default/public interface of the package (which this code is doing).